client/biome.json

```
@@ -1,44 +1,72 @@
+			}
 		}
 	},
 	"javascript": {
 		"formatter": {
 			"quoteStyle": "double"
 		}
-	}
+	},
+	"overrides": [
+		{
+			"includes": ["**/*.svelte", "**/*.astro", "**/*.vue"],
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)